# HG changeset patch
# User Rik <rik@octave.org>
# Date 1343162338 25200
#      Tue Jul 24 13:38:58 2012 -0700
# Node ID 6239f5806c2699a11172e8352887b504d328bec0
# Parent  ff9a638a5555791ce9c4e57bcafbd5c3b38d7e9e
Fix typos introduced by changeset 5d3a684236b0.

* slice.m, uigetfile.m, uiputfile.m: Fix typos introduced by
changeset 5d3a684236b0.

diff --git a/scripts/plot/slice.m b/scripts/plot/slice.m
--- a/scripts/plot/slice.m
+++ b/scripts/plot/slice.m
@@ -99,17 +99,17 @@ function h = slice (varargin)
   elseif (nargs == 7)
     v = varargin{4};
     if (ndims (v) != 3)
       error ("slice: expect 3-dimensional array of values");
     endif
     x = varargin{1};
     y = varargin{2};
     z = varargin{3};
-    if (isvector (x) && isvector (y) && isvector (z)]))
+    if (isvector (x) && isvector (y) && isvector (z))
       [x, y, z] = meshgrid (x, y, z);
     elseif (ndims (x) == 3 && size_equal (x, y, z))
       ## Do nothing.
     else
       error ("slice: X, Y, Z size mismatch");
     endif
     sx = varargin{5};
     sy = varargin{6};
diff --git a/scripts/plot/uigetfile.m b/scripts/plot/uigetfile.m
--- a/scripts/plot/uigetfile.m
+++ b/scripts/plot/uigetfile.m
@@ -76,17 +76,17 @@ function [retfile, retpath, retindex] = 
                defaulttoolkit);
     endif
   endif
 
   if (nargin > 7)
     error ("uigetfile: number of input arguments must be less than eight");
   endif
 
-  defaultvals = {cell (0, 2),        # File Filter
+  defaultvals = {cell(0, 2),         # File Filter
                  "Open File",        # Dialog Title
                  "",                 # Default file name
                  [240, 120],         # Dialog Position (pixel x/y)
                  "off",              # MultiSelect on/off
                  pwd};               # Default directory
 
   outargs = cell (6, 1);
   for i = 1 : 6
diff --git a/scripts/plot/uiputfile.m b/scripts/plot/uiputfile.m
--- a/scripts/plot/uiputfile.m
+++ b/scripts/plot/uiputfile.m
@@ -67,17 +67,17 @@ function [retfile, retpath, retindex] = 
                defaulttoolkit);
     endif
   endif
 
   if (nargin > 3)
     print_usage ();
   endif
 
-  defaultvals = {cell (0, 2),    # File Filter
+  defaultvals = {cell(0, 2),     # File Filter
                  "Save File",    # Dialog Title
                  "",             # Default file name
                  [240, 120],     # Dialog Position (pixel x/y)
                  "create",
                  pwd};           # Default directory
 
   outargs = cell (6, 1);
   for i = 1 : 6
